carbonmark/lib/offsetGetter.ts

```diff
@@ -0,0 +1,11 @@
+import { KlimaRetire } from "@klimadao/lib/types/subgraph";
+
+// Leaves only unique elements from a comma separated list of entries
+export const getOffsetCategories = (offset: KlimaRetire["offset"]) => {
+  return Array.from(new Set(offset.methodologyCategory.split(", "))).join(", ");
+};
+
+// Leaves only unique elements from a comma separated list of entries
+export const getOffsetMainCategory = (offset: KlimaRetire["offset"]) => {
+  return offset.methodologyCategory.split(", ")[0];
+};
```

Comment:
> // Leaves only unique elements from a comma separated list of entries

This is not necessarily true. This function retrieves the first instance of a category, as far as I am aware there is no **main** category.


```suggestion
/** Returns the first category from a list of offset categories. Note: necessary for Retirement PDF category heading **/
export const getOffsetMainCategory = (offset: KlimaRetire["offset"]) => {
  return offset.methodologyCategory.split(", ")[0];
};
```